persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/ReadSideActor.scala

```diff
@@ -100,12 +101,14 @@ private[lagom] class ReadSideActor[Event <: AggregateEvent[Event]](
     case EnsureActive(_) =>
     // Yes, we are active
 
-    case Status.Failure(e) =>
-      throw e
-
     case Done =>
       throw new IllegalStateException("Stream terminated when it shouldn't")
 
+    case Status.Failure(cause) =>
+      // Crash if the globalPrepareTask or the event stream fail
+      // This actor will be restarted by ClusterDistribution
+      throw cause
+
```

Comment:
This was already done for the Scala one, but move it down and added same comment to align with java counterpart.

Reply:
Heh I hadn't even noticed!